compiler/passes/src/type_checking/checker.rs

```diff
@@ -366,42 +366,40 @@ impl<'a> TypeChecker<'a> {
         let check_pedersen_64_bit_input = |type_: &Option<Type>, span: &Span| {
             self.check_type(
                 |type_: &Type| {
-                    matches!(
+                    !matches!(
                         type_,
-                        Type::Boolean
-                            | Type::Integer(IntegerType::I8)
-                            | Type::Integer(IntegerType::I16)
-                            | Type::Integer(IntegerType::I32)
-                            | Type::Integer(IntegerType::I64)
-                            | Type::Integer(IntegerType::U8)
-                            | Type::Integer(IntegerType::U16)
-                            | Type::Integer(IntegerType::U32)
-                            | Type::Integer(IntegerType::U64)
-                            | Type::String
+                        Type::Integer(IntegerType::U128)
+                            | Type::Integer(IntegerType::I128)
+                            | Type::Mapping(_)
+                            | Type::Tuple(_)
+                            | Type::Err
+                            | Type::Unit
                     )
                 },
-                "boolean, integer (up to 64 bits), string".to_string(),
-                type_,
-                *span,
-            );
-        };
-
-        // Helper to check that the type of the argument is a valid input to a Pedersen hash/commit with 128-bit inputs.
-        let check_pedersen_128_bit_input = |type_: &Option<Type>, span: &Span| {
-            self.check_type(
-                |type_: &Type| matches!(type_, Type::Boolean | Type::Integer(_) | Type::String),
-                "boolean, integer, string".to_string(),
+                "address, boolean, field, group, struct, integer, scalar, struct, string".to_string(),
```

Comment:
There might be a discrepancy between this code and the input types documented at https://developer.aleo.org/aleo/opcodes/#hashped64 and https://developer.aleo.org/aleo/opcodes/#hashped128...?

Reply:
The documentation is incorrect and being improved https://github.com/AleoHQ/techops/issues/58